### PR TITLE
Feat/filters

### PR DESCRIPTION
**Pull request by abhij1607**
This PR consist of

- [fix] edit image in post
- [feat] sort

### CODE REVIEW

**Review by shivanigangadharan — APPROVED** [review]
Clean code, good naming conventions used as well.

File: src/utils/sortBy.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+export const sortByTrending = (arr) => {
+  arr.sort((a, b) => {
+    return b.data.likes - a.data.likes;
+  });
+  return arr;
+};
+
+export const sortPosts = (arr, sortOrder) => {
+  switch (sortOrder) {
+    case "OLDEST":
```
Comment (shivampandey0) [inline src/utils/sortBy.js:24]: You can make these cases as Constants to avoid spell mistakes

**Review by shivampandey0 — APPROVED** [review]
Clean code and good naming conventions were used.

good to merge

File: src/utils/sortBy.js (hunk 1 above)
Comment (abhij1607) [reply src/utils/sortBy.js:24]: thanks for the suggestion

**Pull request merged**